src/test/mir-opt/const_prop/optimizes_into_variable.rs

```diff
@@ -119,29 +119,25 @@ fn main() {
 // START rustc.main.SimplifyLocals.after.mir
 // let mut _0: ();
 // let _1: i32;
-// let mut _3: [i32; 6];
```

Comment:
cc @rust-lang/wg-mir-opt Can we block any changes to `mir-opt` tests on #70721 landing? Also, that PR should probably be approved now :rocket:.